transcendence-app/src/shared/local-file/local-file.service.ts

```diff
@@ -76,4 +107,17 @@ export class LocalFileService {
   ): Promise<LocalFile | null> {
     return this.localFileRepository.updateById(id, localFileDto);
   }
+
+  // default values as suggested in lib example
+  async createRandomSVGFile(blocks = 6, width = 100) {
+    const newAvatar = generateAvatar({
```

Comment:
As I have already mentioned, it's kind of horrible to review a PR where half of the local variable types are relying on somehow obscure type inference.

Reply:
Sorry, done :+1: